eclipsecs-sevntu-plugin/src/com/github/sevntu/checkstyle/checks/coding/checkstyle-metadata.properties

```diff
@@ -183,6 +183,9 @@ EitherLogOrThrowCheck.loggingMethodNames = Logging method names separated with c
 WhitespaceBeforeArrayInitializerCheck.name = Whitespace Before Array Initializer
 WhitespaceBeforeArrayInitializerCheck.desc = This checks enforces whitespace before array initializer.
 
+EnumTrailingCommaCheck.name = Enum Trailing Comma
+EnumTrailingCommaCheck.desc = This checks enforces a trailing comma at the end of a line in an enum constant definition.
```

Comment:
https://docs.oracle.com/javase/specs/jls/se8/html/jls-8.html#jls-8.9.1

please remove "line" , this Check looks like care only about "coma after Enum Constant List" only.
If this is not true, than logic need to be changed.